core/src/main/java/io/questdb/cairo/wal/ApplyWal2TableJob.java

```diff
@@ -272,10 +275,18 @@ private void processWalSql(TableWriter tableWriter, WalEventCursor.SqlInfo sqlIn
                 default:
                     throw new UnsupportedOperationException("Unsupported command type: " + cmdType);
             }
+        } catch (CairoException ex) {
+            if (!ex.isCritical() || ex.getErrno() == METADATA_VALIDATION) {
+                LOG.error().$("error applying UPDATE SQL to wal table [table=")
```

Comment:
i would suggest to introduce `ex.isWalTolerable()`. This condition is the same in two places

Reply:
Good idea. Done in 054f31c80d1eac4721cac8a1d6ba1f4e9618cc3e